airflow/hooks/druid_hook.py

```diff
@@ -13,179 +13,71 @@
 # limitations under the License.
 
 from __future__ import print_function
-import logging
-import json
-import time
 
-from pydruid.client import PyDruid
+import logging
 import requests
+import time
 
-from airflow.hooks.base_hook import BaseHook
 from airflow.exceptions import AirflowException
-
-LOAD_CHECK_INTERVAL = 5
-DEFAULT_TARGET_PARTITION_SIZE = 5000000
-
-
-class AirflowDruidLoadException(AirflowException):
-    pass
+from airflow.hooks.base_hook import BaseHook
 
 
 class DruidHook(BaseHook):
-    '''
-    Interact with druid.
-    '''
-
-    def __init__(
-            self,
-            druid_query_conn_id='druid_query_default',
-            druid_ingest_conn_id='druid_ingest_default'):
-        self.druid_query_conn_id = druid_query_conn_id
-        self.druid_ingest_conn_id = druid_ingest_conn_id
+    """
+    Connection to Druid
+    """
+
+    def __init__(self,
+                 conn_id='druid_ingest_default',
+                 timeout=1,
+                 max_ingestion_time=18000):
+        self.conn_id = conn_id
+        self.timeout = timeout
+        self.max_ingestion_time = max_ingestion_time
         self.header = {'content-type': 'application/json'}
 
     def get_conn(self):
-        """
-        Returns a druid connection object for query
-        """
-        conn = self.get_connection(self.druid_query_conn_id)
-        return PyDruid(
```

Comment:
Right now the PyDruid is only used to [construct a connection](https://github.com/apache/incubator-airflow/blob/master/airflow/hooks/druid_hook.py#L47-L62), after building the connection, the PyDruid object is not used in the operator/hook anywhere. Also the `druid_query_conn_id` is not [provided anywhere in the operator](https://github.com/apache/incubator-airflow/blob/master/airflow/operators/hive_to_druid.py). If there is any historical reason to keep this, please let me know. But afaik it is not used in the project.